web/pdf_outline_view.js

```diff
@@ -69,6 +71,51 @@ var PDFOutlineView = (function PDFOutlineViewClosure() {
       };
     },
 
+    /**
+     * Prepend a button before an outline item which allows the user to toggle
+     * the visibility of all outline items at that level.
+     *
+     * @private
+     */
+    _addToggleButton: function PDFOutlineView_addToggleButton(div) {
+      var toggler = document.createElement('div');
+      toggler.className = 'outlineItemToggler';
+      toggler.onclick = function(event) {
```

Comment:
Almost everywhere, except in this file: https://github.com/mozilla/pdf.js/blob/862342189fb1911395d0cc43db083726203c4a88/web/pdf_outline_view.js#L66